File: CoderByte/scala/uno/reverse.scala
    object reverse {
    	def reverse(word: String) = {
    		val res = word.toArray
			for( i <- res.length-1 to 0 by -1){
			    print(res(i));
			}
    	}
	    def main(args: Array[String]) {
	    	reverse("hola")
	    	println ("")
    	}
    }

File: CoderByte/lua/uno/reverse.lua

function reverse( word )
	word_complete=""
	res= string.len(word)
	for i=res,0,-1 do
		word_complete = word_complete..string.sub(word,i,i)
		-- (  ..  ) SE USAN PARA CONCATENAR
		print(string.sub(word,i,i))
	end
	print(word_complete)
end

--mandar a llamar la funcion
reverse("hola")